app/src/main/java/com/dessalines/thumbkey/utils/Utils.kt

```diff
@@ -50,11 +50,11 @@ const val IME_ACTION_CUSTOM_LABEL = EditorInfo.IME_MASK_ACTION + 1
 @Composable
 fun colorVariantToColor(colorVariant: ColorVariant): Color {
     return when (colorVariant) {
-        ColorVariant.SURFACE -> MaterialTheme.colorScheme.surface
-        ColorVariant.SURFACE_VARIANT -> MaterialTheme.colorScheme.surfaceVariant
-        ColorVariant.PRIMARY -> MaterialTheme.colorScheme.primary
-        ColorVariant.SECONDARY -> MaterialTheme.colorScheme.secondary
-        ColorVariant.MUTED -> MaterialTheme.colorScheme.secondary.copy(alpha = 0.5F)
+        ColorVariant.PRIMARY_KEY -> MaterialTheme.colorScheme.primaryContainer
```

Comment:
I don't mind the name changes, but after testing these out, it feels too colorful for a key background, which should be mostly neutral.

Also when I tested this with some themes (the pink theme for instance), the primary container and secondary container were nearly identical, so you couldn't see the separation.

Reply:
Yes, I think you're right. I think I was working on this a bit too late at night and didn't notice a lot of things, I might mark this as a draft and keep working on it for now.

Reply:
No probs!